vj4/ui/components/katex/katex.page.js

```diff
@@ -1,35 +1,9 @@
 import { AutoloadPage } from '../../misc/PageLoader';
 
 const katexPage = new AutoloadPage(async () => {
-  const katex = await System.import('katex');
+  const renderKatex = await System.import('katex/dist/contrib/auto-render.min.js');
   function runKatex($container) {
-    for (const element of $container.find('code')) {
-      const $element = $(element);
-      const text = $element.text();
-      let formula = null;
-      let $outer = null;
-      let isInline = null;
-      if ($element.parent().is('pre') && $element.is('.language-math')) {
-        formula = text;
-        $outer = $element.parent();
-        isInline = false;
-      } else if (text.match(/^\$/) && text.match(/\$$/) && text.length > 2) {
-        formula = text.substr(1, text.length - 2);
-        $outer = $element;
-        isInline = true;
-      } else {
-        continue;
-      }
-      const $wrap = $('<span>');
-      try {
-        katex.render(formula, $wrap[0], {
-          displayMode: !isInline,
-        });
-        $outer.replaceWith($wrap);
-      } catch (ex) {
-        // ignore
-      }
-    }
+    renderKatex($container[0]);
```

Comment:
it accepts a DOM element instead of a jQuery element so a convertion is needed. The jQuery element is built to to keep styles consistent with other "content post-processors".